### PR TITLE
EXAMPLE.md: use docker hub to store example image

### PR DESCRIPTION
**Pull request by liulanzheng**
Signed-off-by: Lanzheng Liu <lanzheng.liulz@alibaba-inc.com>

### CODE REVIEW

File: docs/EXAMPLES.md [hunk 1]
```diff
@@ -39,13 +39,13 @@ And we provide `ctr` subcommand `rpull` as plugin to use feature to support pull
...
 # you will see that the rpull doesn't pull layer data.
-sudo bin/ctr rpull overlaybd-registry.cn-hangzhou.cr.aliyuncs.com/example/redis:6.2.1_obd
+sudo bin/ctr rpull registry.hub.docker.com/overlaybd/redis:6.2.1_obd
 ```
 
 And start a container based on this image.
 
 ```bash
-$ sudo ctr run --net-host --snapshotter=overlaybd --rm -t overlaybd-registry.cn-hangzhou.cr.aliyuncs.com/example/redis:6.2.1_obd demo
+$ sudo ctr run --net-host --snapshotter=overlaybd --rm -t registry.hub.docker.com/overlaybd/redis:6.2.1_obd demo
```
Comment (fuweid) [inline docs/EXAMPLES.md:48]: you can try `docker.io` instead of `registry.hub.docker.com`. it is shorter

Comment (fuweid) [reply docs/EXAMPLES.md:48]: synced with @liulanzheng offline. the overlaybd need to handle 302 http code. keep `registry.hub.docker.com` here

**Review by fuweid — APPROVED** [review]
LGTM

File: docs/EXAMPLES.md (hunk 1 above)
Comment (liulanzheng) [reply docs/EXAMPLES.md:48]: The support for 302 in overlaybd will be added soon

Comment (liulanzheng) [reply docs/EXAMPLES.md:48]: It is not because 302, it is because overlaybd lack of the the support for registry mirrors. We will implement it soon.

**Pull request merged**